### PR TITLE
Add CORE Recommender to Labs page

### PR DESCRIPTION
**Pull request by mhl10**
The following changes have been made in collaboration with @petrknoth and @viktor-yakubiv from the [CORE](https://core.ac.uk) team:
Screen Shot 2020-09-04 at 10 37 00 PM

- This PR also removes the "arXiv HTML5 & Readability" because it is dormant.

### CODE REVIEW

File: labs/_templates/labs/base.html
Comment (viktor-yakubiv) [inline labs/_templates/labs/base.html:16]: You can remove it completely picking up from the git history if needed (if not forget about, what's tricky), or add an additional explanation why this is commented, so you know when this should become uncommented in the future when the commenting condition changed.

**Review by viktor-yakubiv — APPROVED** [review]
Changes look good to me 👍

Just curious, what is the deployment process after merging this: manual waiting for some release or automatic?

**Comment by mhl10** [comment]
> Changes look good to me 👍
> 
> Just curious, what is the deployment process after merging this: manual waiting for some release or automatic?

Right now we deploy labs.arxiv.org manually because changes are so infrequent, but we could easily set this up for continuous deployment on merge.

**Review by SBBCornell — APPROVED** [review]
looks good!

**Pull request merged**
